.github/workflows/rust.yml

```diff
@@ -7,6 +8,8 @@ on:
   pull_request:
     branches:
       - '*'
+  schedule:
+    - cron: '5 4 * * 6'
```

Comment:
That will only run on Sundays (day 6) no?
Don't you want '*' in the last field to run every night?
you chose 5 past 4PM for some particular reason? :-)
https://jasonet.co/posts/scheduled-actions/
e.g. midnight every day would be:
0 0 * * *